pkg/cover/heatmap.go

```diff
@@ -259,17 +259,21 @@ func DoSubsystemsHeatMapStyleBodyJS(
 }
 
 func FormatResult(thm *templateHeatmap, format Format) {
-	thm.Filter(func(row *templateHeatmapRow) bool {
-		if row.IsDir && len(row.Items) > 0 {
-			return true
-		}
-		return slices.Max(row.Covered)-row.Covered[len(row.Covered)-1] >= int64(format.FilterMinCoveredLinesDrop)
-	})
+	if format.FilterMinCoveredLinesDrop > 0 {
```

Comment:
Can you please comment this step?

Reply:
It looks self-explainable to me. In what way does it look confusing?
```type Format struct {``` may be a better place to explain these fields if they look confusing...